packages/angular_devkit/schematics/src/tree/host-tree.ts

```diff
@@ -426,3 +404,17 @@ export class HostTree implements Tree {
     );
   }
 }
+
+export class HostCreateTree extends HostTree {
+  constructor(host: virtualFs.Host) {
```

Comment:
Should be the same host type as it’s superclass (ReadonlyHost<{}>)

Reply:
Done.